voice/bxml/verbs/forward.md

```diff
@@ -6,8 +6,8 @@ Forwards an unanswered incoming call to another number.  Unlike `<Transfer>`, on
-| to                 | Number to forward the call to. Must be in E.164 format (e.g. +15555555555)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| from               | (optional) Number to use for caller ID on the outgoing leg. Must be in E.164 format (e.g. +15555555555). If omitted, assumes the "to" number of the original leg.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| to                 | Number to forward the call to. Must be in E.164 format (e.g. +15555555555).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| from               | (optional) Number to use for caller ID on the outgoing leg. Must be in E.164 format (e.g. +15555555555) or be one of the following acceptable strings: `Restricted`, `Anonymous`, `Private`, or `Unavailable`. If omitted, assumes the "to" number of the original leg.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
"Acceptable" seems redundant here (don't know why GitHub won't allow me to suggest code here)
```
or one of the following: `Restricted`, `Anonymous`, `Private`, or `Unavailable`.
```